src/main/java/frc/robot/commands/MotorTestCommand.java

```diff
@@ -0,0 +1,80 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+
+import frc.robot.subsystems.RollerSubsystem;
+import frc.robot.subsystems.TiltedElevatorSubsystem;
+import frc.robot.subsystems.TiltedElevatorSubsystem.ElevatorState;
+import frc.robot.subsystems.drivetrain.BaseSwerveSubsystem;
+
+public class MotorTestCommand extends InstantCommand {
+    private static final double TEST_DELAY_SECS = 1;
+    private static final double MECH_DELAY_SECS = 2;
+    private static final double SWERVE_DELAY_SECS = 2;
+
+    public MotorTestCommand(BaseSwerveSubsystem swerveSubsystem, TiltedElevatorSubsystem tiltedElevatorSubsystem, RollerSubsystem rollerSubsystem) {
+        super(() -> {
+            // Run tilted elevator to all heights
+            // GROUND - CHUTE - SUBSTATION - CUBEMID - CUBEHIGH - CONEMID - CONEHIGH - SUBSTATION
+            tiltedElevatorSubsystem.setState(ElevatorState.GROUND);
+            Timer.delay(TEST_DELAY_SECS);
+
+            tiltedElevatorSubsystem.setState(ElevatorState.CHUTE);
+            Timer.delay(TEST_DELAY_SECS);
+
+            tiltedElevatorSubsystem.setState(ElevatorState.SUBSTATION);
+            Timer.delay(TEST_DELAY_SECS);
+
+            tiltedElevatorSubsystem.setState(ElevatorState.CUBE_MID);
+            Timer.delay(TEST_DELAY_SECS);
+
+            tiltedElevatorSubsystem.setState(ElevatorState.CUBE_HIGH);
+            Timer.delay(TEST_DELAY_SECS);
+
+            tiltedElevatorSubsystem.setState(ElevatorState.CONE_MID);
+            Timer.delay(TEST_DELAY_SECS);
+
+            tiltedElevatorSubsystem.setState(ElevatorState.CONE_HIGH);
+            Timer.delay(TEST_DELAY_SECS);
+
+            tiltedElevatorSubsystem.setState(ElevatorState.SUBSTATION);
+            Timer.delay(MECH_DELAY_SECS);
+
+            // Run roller in and out
+            for (double i = -1 ; i <= 1; i += .1){
+                rollerSubsystem.setRollPower(i);
+                Timer.delay(0.2);
+            }
+            Timer.delay(TEST_DELAY_SECS);
+
+            // Open motor
+            rollerSubsystem.openMotor();
```

Comment:
later we will have to use the DropCommand on the hall effect branch instead. we can probably run to a whole bunch of positions, end on CONE_MID, and test the dropping there bc it's most complex?